app/src/main/AndroidManifest.xml

```diff
@@ -21,17 +23,21 @@
             android:name="preloaded_fonts"
             android:resource="@array/preloaded_fonts" />
 
-        <activity android:name=".ScanBarcodeActivity"
-            android:parentActivityName=".AddSaleActivity"></activity>
+        <activity
+            android:name=".ScanBarcodeActivity"
+            android:parentActivityName=".AddSaleActivity" />
         <activity android:name=".FillSaleActivity"
             android:parentActivityName=".AddSaleActivity"/>
         <activity android:name=".AddSaleActivity"
             android:parentActivityName=".MainActivity"/>
 
-        <activity android:name=".ListSalesActivity"
-            android:parentActivityName=".MainActivity"></activity>
-        <activity android:name=".LoginActivity"></activity>
+        <activity
+            android:name=".ListSalesActivity"
+            android:parentActivityName=".FilteringBooksActivity" />
+        <activity android:name=".FilteringBooksActivity" />
+        <activity android:name=".SearchBookActivity" />

```

Comment:
You probably want to give parentActivity to everything but the MainActivity

Reply:
Not sure what you mean there, I corrected by adding MainActivity as parent of FilteringBooksActivity, tell me if it is not what you meant. 

I also remove SearchBookActivity which is not used for now (plus we did not agreed on the final architecture of the gui, so it may become obsolete soon)

Reply:
Exactly what I meant, yes :)